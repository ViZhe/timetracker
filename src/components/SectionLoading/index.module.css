
.wrapper {
  display: flex;
  justify-content: center;
  align-items: center;
  margin-top: 5rem;
  margin-bottom: 5rem;
}
