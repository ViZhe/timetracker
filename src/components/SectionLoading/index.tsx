
import { Spin } from 'antd';
import React from 'react';

import styles from './index.module.css';


const SectionLoading: React.FC = () => (
  <div className={styles.wrapper}>
    <Spin />
  </div>
);


export default React.memo(SectionLoading);
