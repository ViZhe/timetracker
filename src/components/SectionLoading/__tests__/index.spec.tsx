
import { shallow } from 'enzyme';
import React from 'react';

import SectionLoading from '../../SectionLoading';


describe('<SectionLoading />', () => {
  it('should match snapshot', () => {
    const component = shallow(<SectionLoading />);

    expect(component).toMatchSnapshot();
  });
});
