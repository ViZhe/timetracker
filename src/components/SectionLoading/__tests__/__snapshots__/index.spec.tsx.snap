// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionLoading /> should match snapshot 1`] = `
<div
  className="wrapper"
>
  <Spin
    size="default"
    spinning={true}
    wrapperClassName=""
  />
</div>
`;
